mnefun/_mnefun.py

```diff
@@ -1589,12 +1590,14 @@ def gen_forwards(p, subjects, structurals, run_indices):
                                 subj + '-trans_head2mri.txt')
                 if not op.isfile(trans):
                     raise IOError('Unable to find head<->MRI trans file')
-            src = op.join(subjects_dir, structurals[si], 'bem',
-                          structurals[si] + '-oct-6-src.fif')
-            if not op.isfile(src):
+            src_space_file = op.join(subjects_dir, structurals[si], 'bem',
+                                     structurals[si] + '-oct-6-src.fif')
+            if not op.isfile(src_space_file):
                 print('  Creating source space for %s...' % subj)
-                setup_source_space(structurals[si], src, 'oct6',
-                                   n_jobs=p.n_jobs)
+                src = setup_source_space(structurals[si], 'oct6',
```

Comment:
Not me. But I also never thought I'd be doing MEG in populations from the recommended default. I'll leave it as is.